opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -117,6 +119,17 @@
      */
     <C> SpanContext extract(Format<C> format, C carrier);
 
+    /**
+     * Closes the Tracer, and flushes the in-memory collection to the configured persistance store.
+     *
+     * <p>
+     * The close method should be considered idempotent; closing an already closed Tracer should not raise an error.
+     * Spans that are created or finished after a Tracer has been closed may or may not be flushed.
```

Comment:
this text implies that while  spans may not be flushed, the tracer must still create them like business as usual, right?

Reply:
Yes, I'd say so (as mentioned in the spec RFC) - an alternative is to NOT allow Span objects to be created (from where? from the builder? from `start`?)